l10n_br_pos/wizard/__init__.py

```diff
@@ -0,0 +1,7 @@
+# © 2016 KMEE INFORMATICA LTDA (https://kmee.com.br)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+# from . import l10n_br_pos_order_return
+# from . import sat_xml_periodic_export
+# from . import stock_invoice_onshipping
+from . import stock_return_picking
```

Comment:
seria melhor remover esses wizards não utilizados do modulo. Se for util, pode rascunho ser aberto uns PR com eles...

Reply:
Acho que seria interessante ir implementando essa questão de acordo com a necessidade que for aparecendo. Por isso vou remover eles do módulo.